tests/plotting/test_plotting.py

```diff
@@ -1667,9 +1673,29 @@ def test_opacity_mismatched_fail(uniform):
     # Test using mismatched arrays
```

Comment:
I can't link to it in the PR because it's too far away, but on line 1666 we have this in `test_opacity_by_array_user_transform`:
```py
    p.show()  # note: =verify_cache_image does not work between Xvfb
```
This is among a bunch of similar opacity-using tests that _do_ have an image regression test. Is this skipped by mistake now? Should we add this back?

Reply:
Added in 9e23560ea330dda75ba379b1984e4bfe61713407 after verifying it works locally with `xvfb`.